src/mpdclient.nim

```diff
@@ -610,18 +612,14 @@ func sortBy*(tag: Tag, descending = false): SortOrder =
 ## Execution
 
 proc runCommand(mpd: MPDClient; cmd: string, args: varargs[string]) =
-  mpd.socket.send cmd
+  var command = cmd
   for arg in args:
-    mpd.socket.send " "
-    mpd.socket.send arg.escape
-  mpd.socket.send "\x0a"
+    command &= " "
+    command &= arg.escape
+  mpd.socket.send command & "\x0a"
 
 proc runCommandOk(mpd: MPDClient; cmd: string, args: varargs[string]) =
```

Comment:
can you make this a template so im sure that its inlined?

Reply:
done